apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -0,0 +1,650 @@
+                await createTestB2BAppAccessRightSet(admin, app)
+            }, 'b2b_app_access_right_set_unique_app')
+        })
+    })
+})
+
+describe('B2BApp permissions for service user', () => {
+    let admin
+    let user
+    let serviceUserClient
+    let serviceUser
+
+    beforeEach(async () => {
+        admin = await makeLoggedInAdminClient()
+        user = await makeClientWithNewRegisteredAndLoggedInUser()
+        const [newServiceUser] = await registerNewServiceUserByTestClient(admin)
+        serviceUser = newServiceUser
+        serviceUserClient = await makeLoggedInClient({ email: newServiceUser.email, password: newServiceUser.password })
+    })
+
+    test('Get all permissions for app', async () => {
+        const [app] = await createTestB2BApp(admin)
+        const [accessRightSet] = await createTestB2BAppAccessRightSet(admin, app)
+        await createTestB2BAppAccessRight(admin, serviceUser, app, accessRightSet)
+
+        const foundRight = await B2BAppAccessRight.getOne(admin, { app: { id: app.id } })
+        expect(foundRight).toHaveProperty('accessRightSet.canReadOrganizations', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canManageOrganizations', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canReadProperties', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canManageProperties', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canReadContacts', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canManageContacts', false)
+    })
+
+    test('Organization', async () => {
+        const [organization] = await registerNewOrganization(user)
+
+        const [app] = await createTestB2BApp(admin)
+        await createTestB2BAppContext(admin, app, organization, { status: 'Finished' })
+        const [right] = await createTestB2BAppAccessRight(admin, serviceUser, app)
+
+        // B2BApp without permissions
+        const countWithoutPermissions = await Organization.count(serviceUserClient, {})
+        expect(countWithoutPermissions).toBe(0)
+        const propertyWithoutPermissions = await Organization.getOne(serviceUserClient, { id: organization.id })
+        expect(propertyWithoutPermissions).toBeUndefined()
+
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await createTestOrganization(serviceUserClient)
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await updateTestOrganization(serviceUserClient, organization.id, {})
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await Organization.softDelete(serviceUserClient, organization.id)
+        })
+
+        const countByUser = await Organization.count(user, {})
+        expect(countByUser).toBe(1)
+
+        // you cannot create a field "canManageOrganizations: true". Only reading! Always false!
```

Comment:
Cannot get it :)

Note says I cannot set true value, but below is false and error is still happening, seems like it's an old description :)